app/models/data_resources/attraction.rb

```diff
@@ -20,21 +20,27 @@ class Attraction < ApplicationRecord
   has_one :operating_company, as: :companyable, dependent: :destroy
   has_many :web_urls, as: :web_urlable, dependent: :destroy
   has_one :external_reference, as: :external, dependent: :destroy
+  has_one :location, as: :locateable, dependent: :destroy
 
   scope :visible, -> { where(visible: true) }
 
   scope :by_category, lambda { |category_id|
     where(categories: { id: category_id }).joins(:categories)
   }
 
+  scope :by_location, lambda { |location_name|
+    where(locations: { name: location_name }).or(where(addresses: { city: location_name }))
+      .left_joins(:location).left_joins(:addresses)
+  }
+
```

Comment:
bin mir unsicher was das left_join hier anders macht, ich hätte sowas in der Art erwartet:

```
where(xxx).joins(:addresses)
```

aber da zickt er gerade mit den :location rum. 
Wenn es so funktioniert, dann prima!

Reply:
same here